src/core/pointcloud/qgseptdecoder.cpp

```diff
@@ -221,6 +221,20 @@ QgsPointCloudBlock *_decompressBinary( const QByteArray &dataUncompressed, const
                                 requestedAttribute.size(), requestedAttribute.type() ) );
   }
 
+  int skippedPoints = 0;
+  QgsPointCloudBlock *block = new QgsPointCloudBlock(
```

Comment:
can you make this a unique_ptr?

Reply:
This is a bit overkill, we're just returning the pointer soon, but ok!